workers/unity/Assets/Gdk/Core/Systems/SpatialOSSendSystem.cs

```diff
@@ -1,16 +1,19 @@
 using System.Collections.Generic;
 using Unity.Entities;
-using UnityEngine;
 
 namespace Improbable.Gdk.Core
 {
     [UpdateInGroup(typeof(SpatialOSSendGroup.InternalSpatialOSSendGroup))]
     public class SpatialOSSendSystem : ComponentSystem
     {
-        internal static class Warnings
+        public struct CustomReplicationSystemAlreadyExists : IErrorContext
```

Comment:
Does this struct need to be `public` ?

Reply:
I think keeping it public allows users to verify whether the error they got in the handler is of type CustomReplicationSystemAlreadyExists.